### PR TITLE
Feature/changes view

### PR DESCRIPTION
**Pull request by Any97Cris**

### CODE REVIEW

File: source/index.blade.php [hunk 1]
```diff
@@ -70,7 +70,7 @@
               </ul>
             </div>
             <div class="ud-hero-image wow fadeInUp" data-wow-delay=".25s">
-              <img src="assets/images/hero/hero-image.svg" alt="hero-image" />
+              <img src="assets/images/hero/print_librecode.png" alt="hero-image" />
```
Comment (vitormattos) [inline source/index.blade.php]: What is hero at this context?

Maybe would be good a new PR  replacing by another name. What you think about this?

But maybe is an unnecessary change.

File: source/index.blade.php [hunk 2]
```diff
@@ -545,7 +549,7 @@ class="ud-contact-form-wrapper wow fadeInUp"
     <!-- ====== Footer End ====== -->
 
     <!-- ====== Back To Top Start ====== -->
-    <a href="javascript:void(0)" class="back-to-top">
+    <a href="javascript:void(0)" class="back-to-top" onclick="backToTop()">
```
Comment (vitormattos) [inline source/index.blade.php]: Look the comment below.

```suggestion
    <a id="back-to-top" class="back-to-top">
```

File: source/index.blade.php [hunk 3]
```diff
@@ -555,6 +559,11 @@ class="ud-contact-form-wrapper wow fadeInUp"
     <script src="assets/js/wow.min.js"></script>
     <script src="assets/js/main.js"></script>
     <script>
+      // ==== back to top
+      backToTop() {
+        window.scrollTo(0, 0);
+      }
```
Comment (vitormattos) [inline source/index.blade.php]: Looking at current site libresign.coop, have a nice effect implementing smooth.

I also suggest to make the JS separated from HTML using ID and adding the onclick event to element.

The code will be a bit bigger than the previous but will separate HTML and JS.

```suggestion
        document.getElementById('back-to-top').onclick = function() {
            e.preventDefault()
            window.scrollTo({
                top: 0,
                left: 0,
                behavior: 'smooth'
            })
        };
```

**Review by vitormattos — APPROVED** [review]
Only small suggestion, but sounds that is good!
Congrats!

File: source/assets/images/hero/print_librecode.png
Comment (giovannism20) [inline source/assets/images/hero/print_librecode.png:1]: Caramba a decisão de imagem ficou ótima!

File: source/index.blade.php (hunk 1 above)
Comment (Any97Cris) [reply source/index.blade.php]: Was created a new issue
